openslides_backend/action/group/create.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict
+
+from ...models.models import Group
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
+from ..register import register_action
+
+
+@register_action("group.create")
+class GroupCreate(CreateAction):
+    """
+    Action to create a group.
+    """
+
+    model = Group()
+    schema = DefaultSchema(Group()).get_create_schema(
+        required_properties=["name", "meeting_id"],
+        additional_optional_fields={
+            "permissions": {"type": "array", "items": {"type": "string"}}
```

Comment:
`permissions` is a valid field in the group model, can't you just pass it in the `optional_properties`?

Reply:
Yes, it can, changed.